spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -548,12 +548,6 @@ def template
 
         @template ||= ProvisioningProfileTemplate.factory(profile_details['template'])
       end
-
-      # @return (String) The name of the template (as displayed in Dev Portal)
-      #   or nil if provisioning profile doesn't have a template
-      def template_name
-        is_template_profile ? template.purpose_display_name : nil
-      end
```

Comment:
So... now I am facing this issue on my environment and found this patch.

However, since on the latest master, this method is used from 2 places (`create_provisioning_profile!` and `repair_provisioning_profile!`,) so, to address the issue, I think instead of removing this method,
rewrite this method behavior or add new method that returns `purpose_name` instead.